HIP/hip-0000-expand-alias-crypto-create-transfer.md

```diff
@@ -0,0 +1,225 @@
+
+Notably, most ledgers in the market utilize the `ECDSA` key for accounts, especially where an `EVM` is concerned. In fact to completely support all Smart Contract scenarios on the `EVM` used by Hedera, an account must possess an alias based on its `ECDSA` public key that dictates the address value observed by teh EVM in transactuions. 
+
+## User stories
+
+1. As a user with an `ECDSA` based account from another chain I would like to create my Hedera account using my `public-address`.
+
+2. As a developer, I would like to `auto-create` a new account using a public key via the `CryptoCreate` transaction.
+
+3. As a developer, I would like to `lazy-create` a new account using a `public-address` via the `CryptoCreate` transaction.
+
+4. As a developer, I would like to `lazy-create` a new account using a `public-address` via the  `CryptoTransfer` transaction.
+
+5. As a developer, I would like to transfer HBAR or tokens to a Hedera account using their `public-address`.
+
+6. As a Hedera user with an `Ethereum-native wallet`, I would like to receive HBAR or tokens in my account by sharing only my `public-address`.
+
+7. As a Hedera user with a `Hedera-native wallet`,  I would like to transfer HBAR or tokens to another account using only the recipient's `public-address`.
+  
+## Specification
+
+This HIP proposes an expansion to the ways in which an account with an alias can be created and referenced to ensure compatibility with users coming from existing chains with `ECDSA` keys and provide greater applicability to smart contract scenarios.
+
+It is thus proposed to
+
+- Add an alias to public key mapping validation check to ensure single user ownership
+- Expand `CryptoCreate` transactions to support setting an alias using a public-key
+- Expand `CryptoTransfer` and `CryptoCreate` transactions to support the input of an alias of the Ethereum account address style with no public key
+
+### Add Public key → alias validation
+
+Additional alias validation logic will be necessary to mandate that at the time of setting an alias it must map to the public key set for the account.
+
+`ED25519` → The alias must match the public key provided/set on the account
+
+`ECDSA` → The alias match the public key or be the public-address of the `ECDSA` key provided/set on the account
+
+### Add  setAlias to `CryptoCreate`
+
+![auto-create-with-key.png](../assets/hip-0000/auto-create-with-key.png)
+
+Add alias bytes property to the `CryptoCreateTransactionBody` [proto file](https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_create.proto)
+
+```proto
+message CryptoCreateTransactionBody {
+	...
+	
+	/**
+   * The bytes to be used as the account's alias. The bytes will be 1 of 2 options. It will be the serialization
+   * of a protobuf Key message for an ED25519/ECDSA primitive key type. If the account is ECDSA based it may also be the public-address, calcluated as the last 20 bytes of the keccak-256 of the ECDSA primitive key.
```

Comment:
Are we going to support ED25519 key aliases for CryptoCreate? See related discussion: https://github.com/hashgraph/hedera-services/pull/3943#discussion_r972471310.

Reply:
Yes. Aliases will remain both simple ED or EC keys